### PR TITLE
cert-rotation: replace RewriteBootstrapHostname with GetBootstrapHost…

### PR DESCRIPTION
**Pull request by vrutkovs**
…name + updated RewriteNodeNames

Followup https://github.com/openshift/library-go/pull/1725

### CODE REVIEW

File: pkg/certs/cert-inspection/certgraphanalysis/collector.go [hunk 1]
```diff
@@ -257,3 +257,45 @@ func MergePKILists(ctx context.Context, first, second *certgraphapi.PKIList) *ce
...
+		return "", err
+	}
+	for _, cm := range etcdConfigMaps.Items {
+		annotation, ok := cm.Annotations["alpha.installer.openshift.io/etcd-bootstrap"]
+		if ok {
+			bootstrapIP = annotation
+			break
+		}
+	}
+
```
Comment (deads2k) [inline pkg/certs/cert-inspection/certgraphanalysis/collector.go:274]: return here if the bootstrapIP is empty?

File: pkg/certs/cert-inspection/certgraphanalysis/collector.go [hunk 2]
```diff
@@ -257,3 +257,45 @@ func MergePKILists(ctx context.Context, first, second *certgraphapi.PKIList) *ce
...
+		if len(descriptionWords) < 1 {
+			continue
+		}
+		bootstrapHostname = descriptionWords[len(descriptionWords)-1]
+		break
+	}
+
+	// Return found bootstrap hostname
+	// If no bootstrap secret found return empty string
+	return bootstrapHostname, nil
```
Comment (deads2k) [inline pkg/certs/cert-inspection/certgraphanalysis/collector.go:300]: return the bootstrap IP too please.

**Comment by deads2k** [comment]
minor comments, but needed prior to merge

/approve

File: pkg/certs/cert-inspection/certgraphanalysis/collector.go (hunk 1 above)
Comment (vrutkovs) [reply pkg/certs/cert-inspection/certgraphanalysis/collector.go:274]: We don't need the bootstrap IP, secret names have k8s name set there ("Peer cert for node ip-10-11-12-13", not "Peer cert for node 10.11.12.13")

Comment (vrutkovs) [reply pkg/certs/cert-inspection/certgraphanalysis/collector.go:281]: Done

**Pull request merged**